src/scss/skin-modern/components/_subtitleoverlay.scss

```diff
@@ -6,14 +6,22 @@
 
   @include hidden;
 
-  bottom: 2em;
+  align-items: flex-end;
+  display: flex;
```

Comment:
Again the question. Did you test in all browsers, especially IE11 (IE11 has flexbox issues)?